app/services/wallets/create_service.rb

```diff
@@ -16,6 +18,8 @@ def create(**args)
       result.wallet = wallet
 
       WalletTransactions::CreateJob.perform_later(
```

Comment:
Shall we create them instantly instead of later ?
I'm wondering cause on wallet creation action, we re-fetch the wallet list right after the create and expect the transactions to be present in the UI.
Or is this line creating them with a pending status and "just" executing them later (creating the "real" transaction) later ?

Reply:
I think it's a different behavior, it's better to have them created async since it's an other business scope, we should definitely think about a websocket :p

Reply:
ok! We'll probably add a disclaimer about potential delay in the UI, either in the wallet page or the creation modal.

Thanks for your answer